### ISSUE TITLE
Test for WCAG failure F91: not correctly marking up table headers

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#158]
http://www.w3.org/WAI/GL/2013/WD-WCAG20-TECHS-20130711/F91

This failure occurs when data tables do not use header elements (th) or other appropriate table mark-up (the scope attribute, headers and id or the ARIA roles columnheader / rowheader) to make the headers programatically determinable from within table content. Making headers programmatically determinable is especially important when data cells are only intelligible together with header information. When screen reader users navigate through the table content horizontally or vertically, the headers that change can be read out to provide the necessary context for the information in the data cells.
## Procedure

For all data tables, check if table headers can be correctly programmatically determined by use of one of the following mechanisms:
- [ ] headers marked up with table header (th) elements, optionally with scope attributes
- [ ] headers and data cells associated using headers and id attributes
- [ ] headers marked up as td elements with the scope attribute
- [ ] headers marked up with ARIA role attributes rowheader or columnheader 
## Expected Results

If all checks 1 - 4 are false, then this failure condition applies and the content fails the Success Criterion.
